MediaAdminBundle/Resources/views/Tree/tree.html.twig

```diff
@@ -0,0 +1,10 @@
+{% import "OpenOrchestraMediaAdminBundle:Tree:tree-macros.html.twig" as macros %}
```

Comment:
If this file is the same that the one used in the backofficeBundle, you don't have to override it